source/hassio/installation.markdown

```diff
@@ -72,16 +73,30 @@ This is the list of packages you need to have available on your system that will
  - ca-certificates
  - curl
  - dbus
- - docker
  - jq
  - network-manager
  - socat
  - software-properties-common
 
-To perform the Hass.io installation, run the following command as root:
+You also need to have Docker-CE installed. There is well-documented procedures for installing Docker on Ubuntu at [Docker.com](https://docs.docker.com/install/linux/docker-ce/ubuntu/).
```

Comment:
This could confuse some users (to think that only ubuntu are supported)
```suggestion
You also need to have Docker-CE installed. There are well-documented procedures for installing Docker at [Docker.com](https://docs.docker.com/install/), you can installation steps for your distro in the menu on the left.
```
I suggest changing the link to be more generic.

Reply:
agreed, correcting...